src/language/constructors.md

```diff
@@ -43,29 +41,35 @@ The `this` keyword refers to the current instance.
 
 ## Initializing formal parameters
 
-The pattern of assigning a constructor argument to an instance variable
-is so common, 
-Dart has initializing formal parameters to make it easy.
+Dart has *initializing formal parameters* to simplify the common pattern of
+assigning a constructor argument to an instance variable. 
+Use `this.propertyName` directly in the constructor declaration,
+and omit the body. 
 
-Initializing parameters can also be used to initialize
+Initializing parameters also allow you to initialize
 non-nullable or `final` instance variables,
-which both must be initialized or provided a default value.
+which both must be initialized or provided a default value:
 
 <?code-excerpt "misc/lib/language_tour/classes/point.dart (constructor-initializer)" plaster="none"?>
 ```dart
 class Point {
   final double x;
   final double y;
 
+  Point(this.x, this.y);
   // Sets the x and y instance variables
   // before the constructor body runs.
-  Point(this.x, this.y);
 }
 ```
 
 The variables introduced by the initializing formals
 are implicitly final and only in scope of the initializer list.
```

Comment:
Since we haven't introduced initializer lists yet, but this paragraph and (now) the next refer to them, can you update this usage to link to [its section](https://dart.dev/language/constructors#initializer-list).

This makes me think we should cover initializer lists earlier, since the named constructors section also references them. I'll leave that to a separate issue issue though and a link will be good enough for now :)

Reply:
Definitely will do. And yes please on the issue! Even just this small amount of content was tricky to rearrange, so that'll definitely need to be another PR 😅